charts/metering-ansible-operator/upstream-values.yaml

```diff
@@ -80,6 +80,10 @@ olm:
 
       * **Configure AWS Billing Data Source** - To assign Pod $$ costs on AWS, create a [read-only IAM user](https://github.com/kube-reporting/metering-operator/blob/master/Documentation/configuring-aws-billing.md) ([example-policy](https://github.com/kube-reporting/metering-operator/blob/master/Documentation/aws/read-only.json)) and [configure Metering](https://github.com/kube-reporting/metering-operator/blob/master/manifests/metering-config/aws-billing.yaml) to use it.
 
+    icon:
```

Comment:
We should put the commit message into perhaps its own `dev-notes.txt` doc in https://github.com/kube-reporting/setup perhaps? Or somewhere devs can find it to be able to repeat this rarely done technique.

Reply:
Yeah that's not a bad idea, but it should be sufficient for now to just refer back to the PR description/commit message for this and emulate the steps listed there.